src/core/src/Context.cpp

```diff
@@ -168,60 +174,31 @@ void Context::reloadStage() {
     fabricResourceManager.setTexturePoolInitialCapacity(getDebugTexturePoolInitialCapacity());
     fabricResourceManager.setDebugRandomColors(getDebugRandomColors());
 
-    // Repopulate the asset registry. We need to do this manually because USD doesn't notify us about
-    // resynced paths when the stage is loaded.
-    const auto stage = UsdUtil::getUsdStage();
+    const auto defaultIonServerPath = pxr::SdfPath("/CesiumServers/IonOfficial");
 
-    // Add sessions first since they can be referenced by tilesets and imagery layers
-    for (const auto& prim : stage->Traverse()) {
-        const auto& path = prim.GetPath();
-        if (UsdUtil::isCesiumIonServer(path)) {
-            SessionRegistry::getInstance().addSession(*_asyncSystem, _httpAssetAccessor, path);
+    // For backwards compatibility. Add default ion server to tilesets without a server.
+    const auto& tilesets = AssetRegistry::getInstance().getAllTilesets();
+    for (const auto& tileset : tilesets) {
+        const auto ionServerPath = tileset->getIonServerPath();
+        if (ionServerPath.IsEmpty()) {
+            const auto tilesetPrim = UsdUtil::getCesiumTileset(tileset->getPath());
+            tilesetPrim.GetIonServerBindingRel().SetTargets({defaultIonServerPath});
```

Comment:
Is it worth notifying the users the file is being altered? Our customers and OV users in general are pretty early adopters, so it's arguable that a release not vs something in-app is sufficient.

Also, should there be an issue to remove this in some number of months in the future?

Reply:
Since we're pre-release, IMO a console message seems sufficient. v1.0 seems like a good time to remove the backwards compatibility.

Reply:
Added warning in https://github.com/CesiumGS/cesium-omniverse/pull/594/commits/fb181ccbab57d371da9a5a9cc9e6b10739ada315

Reply:
Issue here: https://github.com/CesiumGS/cesium-omniverse/issues/598